ddp-workspace/projects/ddp-testboston/src/app/components/user-registration-prequal/user-registration-prequal.component.ts

```diff
@@ -22,13 +23,25 @@ export class UserRegistrationPrequalComponent implements OnInit, OnDestroy {
   public isLoading = false;
   private anchor: Subscription;
   @ViewChild('captcha', { static: false }) private captcha: RecaptchaComponent;
+  private subscription: Subscription;
```

Comment:
Could we create Subscription class instance:
`private anchor = new Subscription();`
and add subscriptions using:
`this.anchor.add(sub);`
instead of creating two subscription management properties in the component.

Reply:
0f778a11cd7c9b457ca69e85c3f8b2f212e9b994